src/vmm/src/guest_config/templates/test_utils/mod.rs

```diff
@@ -0,0 +1,227 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Looks like this content was not removed from `guest_config/templates/*`. I expected this to be a move operation as the commit message suggests, not a copy.

Reply:
You are correct. I think the test data was introduced following a rebase that I didn't check through. I will remove from the non-test modules.